core/bin/genesis_generator/src/main.rs

```diff
@@ -80,7 +82,10 @@ async fn generate_new_config(
 
     let base_system_contracts = BaseSystemContracts::load_from_disk().hashes();
     let mut updated_genesis = GenesisConfig {
-        protocol_version: Some(ProtocolVersionId::latest() as u16),
+        protocol_version: Some(ProtocolSemanticVersion {
+            minor: ProtocolVersionId::latest(),
+            patch: 0.into(),
```

Comment:
Probably worth adding a comment on why we don't care about the patch version being correct

Reply:
in genesis generator there is no "correct" patch version, but I will add a comment